### DISCUSSION TITLE
Max Sharpe portfolio with risk parity constraint

### DISCUSSION THREAD

**Discussion by ghost** [dcajasn/Riskfolio-Lib#87] [Q&A] [answered]
I want to find portfolio which maximizes Sharpe under the constraint that each asset contributes with the same amount of risk. Is it possible to do with riskfolio please?

**Comment by dcajasn** [comment, marked as answer]
Hi @qmgm 

No, and I think that this problem can't be posed as a DCP problem.

Best,
Dany
